src/components/Reports/FourteenMonthReports/Layout/Table/Table.test.tsx

```diff
@@ -169,57 +102,86 @@ const router = {
 };
 const onRequestSort = jest.fn();
 
-interface ComponentsProps {
+interface TestComponentProps {
   orderBy?: OrderBy | null;
 }
-const Components: React.FC<ComponentsProps> = ({ orderBy = 'name' }) => {
-  const mocks = transformedData();
-  const getTotals = () => mocks?.currencyGroups[0].totals.months || [];
 
+const InnerComponent: React.FC<Pick<TestComponentProps, 'orderBy'>> = ({
```

Comment:
This `Pick` doesn't do anything because `TestComponentProps` only has an `orderBy` key.

```suggestion
const InnerComponent: React.FC<TestComponentProps> = ({
```

Reply:
I guess since TestComponentProps is used twice in this file it's best to use the props interface

Reply:
`Pick` is a great choice when you only want some of the props. But since you want all of the props here (even though it's just one), it seems good to keep things simple and use `TestComponentProps`.